### PR TITLE
Added qps probe script for Predefined Workflow sock-shop 

### PR DESCRIPTION
**Pull request by oumkale**
**What this PR does / why we need it**:

Added cmd-probe support feature for sock-shop QPS 

**Special notes for your reviewer**:
Screenshot from 2021-01-12 16-52-43

### CODE REVIEW

File: custom/git-app-deployer/README.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+### For -timeout
+
+Timeout is used for termination of application. The exceeding time by default is 300s.
+You may change the default time value e.g
+
+- -timeout=400
+
+### How the git app deployer works in the workflow:
+A kubeconfig file is a file used to configure access to Kubernetes when used in conjunction with the kubectl command line tool (or other clients).
```
Comment (ksatchit) [inline custom/git-app-deployer/README.md:39]: Do we need this info>?

File: custom/git-app-deployer/README.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+It creates a namespace and then installs the required application on the basis of the given -namespace and -filepath.
+
+If namespace is already in exist then it shows log and start installing sock-shop.
```
Comment (ksatchit) [inline custom/git-app-deployer/README.md:42]:
```suggestion
If namespace is already exists then it shows log and start installing sock-shop.
```

File: custom/git-app-deployer/README.md (hunk 1 above)
Comment (oumkale) [reply custom/git-app-deployer/README.md:39]: Not such I am removing it!

**Comment by oumkale** [comment]
Suggested changes are made. PTAL again @ispeakc0de.

File: custom/git-app-deployer/app-test/main.go [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+			vars.totalQueries -= front.Value.(int)
+			vars.qpsValue = strconv.Itoa(100 * (vars.totalQueries / timeInterval))
+		}
+		log.Infof("[Status]: Current total requests : ", req)
+		log.Infof("[Status]: Current QPS value is   : ", vars.qpsValue)
+		time.Sleep(1 * time.Second)
+	}
+}
+
+//GetRequests will fetch the responce from metrics and calculate the total requests from front-end of sock-shop.
```
Comment (ispeakc0de) [inline custom/git-app-deployer/app-test/main.go:100]: responce -> response

**Review by ispeakc0de — DISMISSED** [review]

File: custom/git-app-deployer/app-test/main.go (hunk 3 above)
Comment (ispeakc0de) [inline custom/git-app-deployer/app-test/main.go:100]:
```suggestion
//GetRequests will fetch the response from metrics and calculate the total requests from front-end of sock-shop.
```

**Review by ispeakc0de — APPROVED** [review]

**Review by ispeakc0de — APPROVED** [review]

**Review by ksatchit — APPROVED** [review]

**Pull request merged**